colibre/scripts/deadtime_evolution.py

```diff
@@ -0,0 +1,90 @@
+"""
+Plots dead time fraction v.s. step size
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from swiftpipeline.argumentparser import ScriptArgumentParser
+from glob import glob
+import re
+import scipy.stats as stats
+
+arguments = ScriptArgumentParser(
+    description="Creates a run performance plot: evolution of the dead time fraction in scale factor bins"
+)
+
+run_names = arguments.name_list
+run_directories = [f"{directory}" for directory in arguments.directory_list]
+output_path = arguments.output_directory
+
+plt.style.use(arguments.stylesheet_location)
+
+fig, ax = plt.subplots()
+
+color_index = 0
```

Comment:
Why not use `enumerate()` for `color_index`?

Reply:
Because I copied this script from `stepsize_deadtime.py`, where it is also done like that. Now that I think about it, I also wrote that script, and you did not review that one. So that is probably why.